index.js

```diff
@@ -1,8 +1,13 @@
 const grpc = require("grpc");
 const protoLoader = require("@grpc/proto-loader");
 
+function getNestedObject(nestedObj, pathArr) {
```

Comment:
I think a more appropriate name of this cool function is `getProtoFromPackageDefinition` because the role of this function is to get proto object from a package definition whether nested or not. And, because of the role, it is more expressive to receive `packageName` and split by dot rather than pre-splitted `pathArr`.

like follows:

```js
getProtoFromPackageDefinition(packageDefinition, packageName){
  const pathArr = packageName.split(".");
  ...
}
```

What do you think?

Reply:
Indeed, that's better 👍 I have updated the PR.